binding/Binding/GRVkBackendContext.cs

```diff
@@ -1,37 +1,87 @@
 ﻿using System;
+using System.Runtime.InteropServices;
 
 namespace SkiaSharp
 {
-	public class GRVkBackendContext : SKObject, ISKReferenceCounted
+#if THROW_OBJECT_EXCEPTIONS
+	using GCHandle = SkiaSharp.GCHandleProxy;
+#endif
+
+	public class GRVkBackendContext : IDisposable
 	{
-		[Preserve]
-		internal GRVkBackendContext (IntPtr h, bool owns)
-			: base (h, owns)
+		private GRVkGetProcDelegate getProc;
+		private GRVkGetProcProxyDelegate getProcProxy;
+		private GCHandle? getProcHandle;
+		private IntPtr getProcContext;
+
+		public IntPtr VkInstance { get; set; }
+
+		public IntPtr VkPhysicalDevice { get; set; }
+
+		public IntPtr VkDevice { get; set; }
+
+		public IntPtr VkQueue { get; set; }
+
+		public UInt32 GraphicsQueueIndex { get; set; }
+
+		public UInt32 MaxAPIVersion { get; set; }
+
+		public GRVkExtensions Extensions { get; set; }
+
+		public IntPtr VkDeviceFeatures { get; set; }
+
+		public IntPtr VkDeviceFeatures2 { get; set; }
+
+		public GRVkGetProcDelegate GetProc {
```

Comment:
The only thing I have a question about is what we should call this property... `GetProc` is fine, but is there a .NET-ism for this? I'm going to have to browse the reference sources...

Reply:
Well, this is how this is called in Skia. I would rather not diverge too much, especially in such niche API.